appveyor.yml

```diff
@@ -0,0 +1,106 @@
+version: "{build}"
+clone_depth: 1
+os: Visual Studio 2022
+platform: x64
+test: false
+environment:
+  SIGN_PATH_TOKEN:
+    secure: w9Tr3wEpZP53yt8gtJ0G+/hUbjJYuyyPjtEGqEgUOeFYyLM0fqoC/OWkJnRkqhgt
+init:
+  - ps: $env:BUILD_YYYYMMDD=(Get-Date).ToString("yyyyMMdd")
+install:
+  - appveyor DownloadFile https://win.rustup.rs/ -FileName rustup-init.exe
+  - rustup-init -yv --default-toolchain stable-x86_64-pc-windows-msvc
+  - set PATH=%PATH%;%USERPROFILE%\.cargo\bin
+  - rustup target add i686-pc-windows-msvc
+before_build:
+  - git submodule update --init --depth 1
+build_script:
+  - cmake -B x86 -A Win32 -DBUILD_TESTING=OFF
+  - cmake --build x86 --config Release
+  - cmake -B x64 -A x64 -DBUILD_TESTING=OFF
+  - cmake --build x64 --config Release
+  - mkdir dist
+  - mkdir nsis
+  - copy installer\* nsis\
+  - copy COPYING.txt nsis\
+  - mkdir nsis\Dictionary
+  - copy x64\libchewing\data\*.dat nsis\Dictionary\
+  - mkdir nsis\x86
+  - copy x86\ChewingTextService\Release\*.dll nsis\x86\
+  - copy x86\libchewing\Release\*.dll nsis\x86\
+  - copy x86\ChewingPreferences\Release\*.exe nsis\
+  - copy x86\libchewing\chewing-cli.exe nsis\
+  - mkdir nsis\x64
+  - copy x64\ChewingTextService\Release\*.dll nsis\x64\
+  - copy x64\libchewing\Release\*.dll nsis\x64\
+  - cd nsis
+  - makensis installer.nsi
+
+for:
+  - branches:
+      only:
+        - nightly
+    before_build:
+      - git submodule update --init --remote --depth 1
```

Comment:
Is it required? I mean is this duplicate with line 17?

Reply:
Note the `--remote`. This overrides the default (line 17) so the nightly build fetches latest libchewing.